contrib/lxqt-panel/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "lxqt-panel"
+pkgver = "2.0.1"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCPULOAD_PLUGIN=NO",
+    "-DNETWORKMONITOR_PLUGIN=NO",
+    "-DSYSSTAT_PLUGIN=NO",
+    "-DALSA_PLUGIN=NO",
+    "-DVOLUME_USE_ALSA=NO",
+]
+hostmakedepends = [
+    "cmake",
+    "lxqt-build-tools",
+    "ninja",
+    "pkgconf",
+    "perl",
+]
+makedepends = [
+    "kwindowsystem-devel",
+    "layer-shell-qt-devel",
+    "libdbusmenu-lxqt-devel",
+    "liblxqt-devel",
+    "libsensors-devel",
+    "lxqt-globalkeys-devel",
+    "lxqt-menu-data",
+    "qt6-qttools-devel",
+    "solid-devel",
+    "xcb-util-devel",
+    "libpulse-devel",
```

Comment:
```suggestion
    "kwindowsystem-devel",
    "layer-shell-qt-devel",
    "libdbusmenu-lxqt-devel",
    "liblxqt-devel",
    "libpulse-devel",
    "libsensors-devel",
    "lxqt-globalkeys-devel",
    "lxqt-menu-data",
    "qt6-qttools-devel",
    "solid-devel",
    "xcb-util-devel",
```

(sort all the lists)